app/helpers/journals_helper.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+module JournalsHelper
+  def journal_status_options(_journal)
+    [[' ', nil], ['Succeeded, no errors', 'succeeded'], ['Succeeded, with errors', 'succeeded_errors'], ['Failed', 'failed']]
```

Comment:
Can we use locales for labels? Maybe using [activerecord nested attributes translations](https://guides.rubyonrails.org/i18n.html#translations-for-active-record-models) fit this case